src/elm/clinical.ts

```diff
@@ -28,10 +30,10 @@ export class ValueSetRef extends Expression {
   name: string;
   libraryName: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.ValueSetRef) {
     super(json);
-    this.name = json.name;
-    this.libraryName = json.libraryName;
+    this.name = json.name || '';
+    this.libraryName = json.libraryName || '';
```

Comment:
Same deal as above w/r/t the [logical specification](https://cql.hl7.org/04-logicalspecification.html#valuesetref). `name` and `libraryName` should be `name?: string` and `libraryName?: string` respectively, and these default empty strings can be removed. My bad again!